content/atom.md

```diff
@@ -0,0 +1,8 @@
+title = "Atom Syndication"
+description = "This is the Atom XML file"
+date = "2021-12-29T22:36:33Z"
+template = "atom"
+content_type = "application/atom+xml"
```

Comment:
TIL this is the correct content type for Atom feeds.
I've been using `application/rss+xml` for mine for the longest time, which _is_ still valid, but this one explicitly identifies an Atom feed.